.travis.yml

```diff
@@ -6,6 +6,7 @@ julia:
     - 0.7
     - 1.0
     - 1.1
+	- 1.3
```

Comment:
Curiously, Travis didn't run. Might this flagged whitespace be the reason? (Maybe you are using tabs and travis expects spaces?)